prow/config/topology/external-istiod.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "kind": "Kubernetes",
+    "clusterName": "external",
+    "podSubnet": "10.10.0.0/16",
+    "svcSubnet": "10.255.10.0/24",
+    "network": "network-1",
+    "primaryClusterName": "external",
+    "configClusterName": "config"
+  },
+  {
+    "kind": "Kubernetes",
+    "clusterName": "config",
+    "podSubnet": "10.20.0.0/16",
+    "svcSubnet": "10.255.20.0/24",
+    "network": "network-1",
+    "primaryClusterName": "external",
+    "configClusterName": "config",
+    "meta": {
+      "fakeVM": false
```

Comment:
What's this? Do we need it?
cc @stevenctl

Reply:
I would put this on the `remote` cluster.

Reply:
And maybe on the `external` cluster or this `cluster` as well. It only makes sense to have the fake VM for the "primary" or "config".